### DISCUSSION TITLE
Add "ServerCreate" Websocket event.

### DISCUSSION THREAD

**Discussion by abdulrahman1s** [stoatchat/discussions#43] [Feature Suggestions]
I looked into the current ws [events](https://developers.revolt.chat/websockets/events), and I was surprised that "ServerCreate" does not exists.. it really needed in many cases.

**Comment by insertish** [comment]
Currently that job is pretty much down to `ServerMemberJoin`, instead of sending both `Create` and `Join` when a new guild is created, only `Join` is sent and hence if the ids of the packet user and client user match, the client knows it just joined a new guild.

Is there any specific use case it's needed for?

**Reply by abdulrahman1s** [reply]
I think it's backend job..
Yeah I can detect whatever server created with `ServerMemberJoin` event.. But it depends on cache
\+ It would be nice not to make another rest request to fetch the target server

**Reply by insertish** [reply]
Created an issue here: https://github.com/revoltchat/delta/issues/32
Locking thread.
